src/main/webapp/app/tasks/filter-pipe/filter-task.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Task } from '../task';
+
+@Pipe({
+  name: 'filterTask',
+  pure: false
+})
+export class FilterTaskPipe implements PipeTransform {
+
+  transform(tasks: Task[], text: string): Task[] {
+    if (text) {
```

Comment:
will trigger also on whitespaces, should check for empty strings

Reply:
Right, fixed